doc/howtos/rdtraining/08_relations.rst

```diff
@@ -0,0 +1,267 @@
+    name                      Char                      required
+    ========================= ========================= =========================
+
+    - Add the menus as displayed in the **Goal**
+    - Add the field ``tag_ids`` on your ``estate.property`` model and in its form and tree views
+
+    Tip: in the view, use the ``widget="many2many_tags"`` attribute as done
+    `here <https://github.com/odoo/odoo/blob/5bb8b927524d062be32f92eb326ef64091301de1/addons/crm_iap_lead_website/views/crm_reveal_views.xml#L36>`__.
+    The ``widget`` attribute will be explained in :ref:`a later part <howto/rdtraining/12_sprinkles>`.
+    detail. For now, you can try to add or remove it and see the result ;-)
+
+One2many
+========
+
+**Reference**: the documentation related to this topic can be found in
+:class:`~odoo.fields.One2many`.
+
+.. note::
+
+    **Goal**: at the end of this section:
+    
+    - A new ``estate.property.offer`` model is created with the corresponding form and tree view.
+    - Offers are added to the ``estate.property`` model:
+
+    .. image:: 08_relations/media/property_offer.png
+        :align: center
+        :alt: Property offers
+
+In our real estate module, we want to define the concept of property offer. A property offer
+is an amount a potential buyer offers to the seller. The offer can be lower or higher than the
+expected price.
+
+An offer applies to **one** property, but the same property can have **many** offers:
+the concept of **many2one** appears once again. However, in this case we want to display the list
+of offers for a given property: we will use the **one2many** concept.
+
+A one2many is the inverse of a many2one. For example, we defined
+on our test model a link to the ``res.partner`` model thanks to the field ``partner_id``.
+We can define the inverse relation, i.e. the list of test models linked to our partner::
+
+    test_ids = fields.One2many("test.model", "partner_id", string="Tests")
+
+The first parameter is called the ``comodel``, while the second parameter is the field we want to
+inverse.
+
+By convention, one2many fields have the ``_ids`` suffix. They behave as a list of records, meaning
+that accessing the data must be done in a loop::
+
+    for test in partner.test_ids:
+        print(test.name)
+
+.. danger::
+
+    Because a :class:`~odoo.fields.One2many` is a virtual relationship,
+    there *must* be a :class:`~odoo.fields.Many2one` field defined in the comodel.
+
+.. exercise:: Add the Real Estate Property Offer table
+
+    - Create the ``estate.property.offer`` model and add the following fields:
+
+    ========================= ================================ ============= =================
+    Field                     Type                             Attributes    Values
+    ========================= ================================ ============= =================
+    price                     Float
```

Comment:
The psolution](https://github.com/odoo/technical-training-solutions/blob/14.0-core/estate/models/estate_property_offer.py#L23) has a `required=True`; who is right?